language/diem-tools/writeset-transaction-generator/src/main.rs

```diff
@@ -69,24 +117,64 @@ fn main() -> Result<()> {
             execute_as,
         ),
         Command::CreateDiemFrameworkRelease { url, version } => {
-            let remote = JsonRpcDebuggerInterface::new(url.as_str())?;
-            let remote_modules = remote.get_diem_framework_modules_by_version(version)?;
-            let local_modules = build_stdlib().into_iter().map(|(_, m)| m).collect();
-            let payload = create_release_writeset(remote_modules, local_modules)?;
-            verify_payload_change(&remote, Some(version), &payload)?;
+            if let Ok(artifact) = load_artifact() {
+                if artifact.remote_json_rpc_url != url || artifact.version > version {
```

Comment:
I think this will fail if we try to create releases for multiple deployments from the same git branch. We probably want `load_artifact` to take a `chain_id` or `url` + have separate directories for release artifacts of each deployment.

Reply:
That's right. Lemme change that